apps/Arm/vm_virtio_net/CMakeLists.txt

```
@@ -43,6 +57,18 @@ if(VIRTIO_NET_PING)
         "etc/init.d"
         overlay
     )
+
+    # The ZCU102 Filesystem expects init scripts to be setup like this, else they
+    # won't run
+    if("${KernelARMPlatform}" STREQUAL "zcu102")
+        AddLinkToOverlayDir("K90ping_test" "../init.d/S90ping_test" "etc/rc0.d/" overlay)
```

We could use a loop here to avoid the redundancy
```suggestion
        foreach(i RANGE 5) # run at rc0 - rc5
            AddLinkToOverlayDir("K90ping_test" "../init.d/S90ping_test" "etc/rc${i}.d/" overlay)
        endforeach()            
```

---

ok, to be investigated later then why this fails.